server/lib/nl/scraper.py

```diff
@@ -190,6 +190,18 @@
     }]
 }
 
+# Max tries to wait for all svgs to be drawn on a subject page
+_MAX_SVG_WAIT_TRIES = 10
```

Comment:
How much e2e delay does this translate to?

Reply:
by delay, do you mean extra waiting time even though everything is already drawn? In that case, the max delay would be 0.5s

Reply:
I mean e2e time for all charts to render.  5s?

Reply:
e2e time wasn't affected by this _MAX_SVG_WAIT_TRIES constant .. but either way, the constant is gone now & e2e time for all charts to render depends on the query but most ended up taking around 5s.